src/utils/index.ts

```diff
@@ -0,0 +1,8 @@
+function FormatString(str: string, ... val: string[]):string {
```

Comment:
Why is this function necessary?

Reply:
it's for localization / string mapping stuff I think, like you would write error messages with "{id}" for formatting

Reply:
It might have use in the future, but I agree that it's not necessary at the moment.